backend/Dockerfile

```diff
@@ -52,17 +50,20 @@ RUN pip install --no-cache psycopg[binary] gunicorn
 # copy project
 COPY --chown=app:app . $APP_HOME
 
-# change to the app user
-USER app
+# # change to the app user
+# USER app
```

Comment:
Why do you need to run the application as root?

Reply:
Great question. I was changing a lot trying to get things to work, so I'll have to double check if this was necessary.

Reply:
ok